platform/jewel/buildSrc/src/main/kotlin/org/jetbrains/jewel/buildlogic/theme/IntUiThemeDescriptorReader.kt

```diff
@@ -7,22 +7,24 @@ import com.squareup.kotlinpoet.KModifier
 import com.squareup.kotlinpoet.PropertySpec
 import com.squareup.kotlinpoet.TypeSpec
 import com.squareup.kotlinpoet.joinToCode
-import java.time.ZoneOffset
-import java.time.ZonedDateTime
 import kotlinx.serialization.json.JsonObject
 import kotlinx.serialization.json.JsonPrimitive
 
 internal object IntUiThemeDescriptorReader {
-
     private val colorGroups = setOf("Gray", "Blue", "Green", "Red", "Yellow", "Orange", "Purple", "Teal")
     private val colorClassName = ClassName("androidx.compose.ui.graphics", "Color")
 
-    fun readThemeFrom(themeDescriptor: IntellijThemeDescriptor, className: ClassName, sourcePath: String) =
+    fun readThemeFrom(
+        themeDescriptor: IntellijThemeDescriptor,
+        className: ClassName,
+        sourcePath: String,
+        ijpMajor: String,
+    ) =
         FileSpec.builder(className)
             .apply {
                 indent("    ")
-                val todayUtc = ZonedDateTime.now().withZoneSameInstant(ZoneOffset.UTC).toLocalDate()
-                addFileComment("Generated by the Jewel Int UI Palette Generator on $todayUtc\n")
+                addFileComment("Generated by the Jewel Int UI Palette Generator for IJP $ijpMajor\n")
+                addFileComment("*** DO NOT EDIT MANUALLY! Use generateThemes task to update. ***\n")
```

Comment:
This change was made so that the generated file is more stable and does not require re-committing every day :)